public/css/styles.css

```diff
@@ -2,6 +2,7 @@
 width : 500 px;
 margin: 0 auto;
```

Comment:
Not responsive
![image](https://user-images.githubusercontent.com/5979667/68602202-ef6d2080-04cb-11ea-95b5-41bf7ab063e4.png)

Reply:
.columns could be
    width: 15vw;
    height: 15vh;